lib/types.d.ts

```diff
@@ -213,6 +215,8 @@ declare module "node-sql-2" {
 		descending:OrderByValueNode
 		asc:OrderByValueNode
 		desc:OrderByValueNode
+		existsKeyElement(str:string):BinaryNode
```

Comment:
These two functions should accept only the result from the array function (`BinaryNode`).